tasty-plutus/src/Test/Tasty/Plutus/Script/Property.hs

```diff
@@ -210,6 +213,75 @@ scriptPropertyPass ::
   WithScript p ()
 scriptPropertyPass = mkScriptPropertyWith OutcomeAlwaysPass
 
+{- | Given a 'Generator' containig a way to generate a seed and function
+ to create 'TestItems' as well as a function to create "TestScript' from the seed,
+ check that:
+
+ * For any 'TestItems' with Outcome equals 'Pass', the 'TestScript' succeeds; and
+ * For any 'TestItems' with Outcome equals 'Fail', the 'TestScript' fails.
```

Comment:
'Outcome' is an identifier, so it should be in single quotes. It's also more natural to phrase it like:

For any 'TestItems' where the 'Outcome' is 'Pass', ...